### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by adith-a-danthi**
Added landing page UI

### CODE REVIEW

**Review by prachi-sahani — COMMENTED** [review]
overall the code looks good.

suggestion - if you have already reset the browser styles in component library, you need not reset it here again

File: css/base.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  right: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.ecomm .card-img {
+  height: 450px;
+  max-height: 450px;
```
Comment (ZuberDunge) [inline css/base.css:34]: if you're giving max height 450, then why max height 450px again?

**Review by ZuberDunge — APPROVED** [review]
Everything looks fine. Good work Adith.

File: css/base.css (hunk 1 above)
Comment (adith-a-danthi) [reply css/base.css:34]: Overriding component library styles

**Pull request merged**